### PR TITLE
OV-153: Show avatars in video menu

### PR DESCRIPTION
**Pull request by o-nedashkivska**
Related issue - https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/issues/153

This PR adds avatars to avatars menu.

Screenshot 2024-09-04 at 12 33 18
Screenshot 2024-09-05 at 16 39 03

### CODE REVIEW

**Review by JKaypa — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/studio/components/video-menu/components/menu-content/script-content/components/script.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import { Editable, EditablePreview, EditableTextarea } from '@chakra-ui/react';
+
+import {
+    Icon,
+    IconButton,
+    VStack,
+} from '~/bundles/common/components/components.js';
+import { useAppDispatch, useCallback } from '~/bundles/common/hooks/hooks.js';
+import { IconName } from '~/bundles/common/icons/icon-name.js';
```
[the remark this answers is not in the document]
Comment (o-nedashkivska) [reply frontend/src/bundles/studio/components/video-menu/components/menu-content/script-content/components/script.tsx]: Done

File: frontend/src/bundles/studio/components/video-menu/components/menu-content/script-content/script-content.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import { IconName } from '~/bundles/common/icons/icon-name.js';
+import { actions as studioActions } from '~/bundles/studio/store/studio.js';
+
+import { Script } from './components/script.js';
+import { NEW_SCRIPT_TEXT } from './constants/constants.js';
+
+const ScriptContent: React.FC = () => {
+    const dispatch = useAppDispatch();
+    const { scripts } = useAppSelector(({ studio }) => ({
+        scripts: studio.scripts,
```
[the remark this answers is not in the document]
Comment (o-nedashkivska) [reply frontend/src/bundles/studio/components/video-menu/components/menu-content/script-content/script-content.tsx]: Done

**Review by sergiy4 — APPROVED** [review]

File: backend/src/bundles/avatars/config/config.ts [hunk 3]
```diff
@@ -1,5 +1,8 @@
 import avatars from './avatar.config.json';
 
+// Removed styles "technical-standing" and "graceful-standing" from "lisa" avatar
+// due to broken images and insufficient config
+
```
Comment (anton-otroshchenko) [inline backend/src/bundles/avatars/config/config.ts:5]: Please remove it

**Review by anton-otroshchenko — APPROVED** [review]

File: backend/src/bundles/avatars/config/config.ts (hunk 3 above)
Comment (o-nedashkivska) [reply backend/src/bundles/avatars/config/config.ts:5]: @nikita-remeslov said that I should leave a comment why I deleted some avatars

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**